internal/pkg/json/schema/schema.go

```diff
@@ -9,12 +9,14 @@ import (
 
 	"github.com/santhosh-tekuri/jsonschema/v5"
 
+	"github.com/keboola/keboola-as-code/internal/pkg/filesystem"
 	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/naming"
 	"github.com/keboola/keboola-as-code/internal/pkg/utils"
 	"github.com/keboola/keboola-as-code/internal/pkg/utils/orderedmap"
 )
 
-func ValidateSchemas(objects model.ObjectsProvider) error {
+func ValidateSchemas(objects *model.State) error {
```

Comment:
`model.ObjectsProvider` je interface pre `state.State`, kedze tu vznikali cyklicke zavislosti.

No `model.ObjectsProvider` odkazuje teraz uz na `naming`,  `naming` potrebuje model.

Preto som tento subor fixol, a uz nie je zavisly na `naming`.
V dalsich PRs to bude vyriesene lepsie.